src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,1106 @@
+    });
+  });
+  describe('additionalData', () => {
+    it('has valid structure', () => {
+      const submissionObj = JSON.parse(transform(null, inputObject));
+      expect(submissionObj).haveOwnProperty('additionalData');
+      expect(submissionObj.additionalData).to.be.an('object');
+      expect(submissionObj.additionalData).haveOwnProperty(
+        'additionalComments',
+      );
+      expect(submissionObj.additionalData).haveOwnProperty('bankruptcy');
+      expect(submissionObj.additionalData.bankruptcy).to.be.an('object');
+    });
+    it('has valid data', () => {
+      const submissionObj = JSON.parse(transform(null, inputObject));
+      expect(submissionObj.additionalData.additionalComments).to.equal(
+        'Supporting personal statement...',
+      );
+    });
+    describe('bankruptcy', () => {
+      it('has valid structure', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(submissionObj.additionalData.bankruptcy).haveOwnProperty(
+          'hasBeenAdjudicatedBankrupt',
+        );
+        expect(submissionObj.additionalData.bankruptcy).haveOwnProperty(
+          'dateDischarged',
+        );
+        expect(submissionObj.additionalData.bankruptcy).haveOwnProperty(
+          'courtLocation',
+        );
+        expect(submissionObj.additionalData.bankruptcy).haveOwnProperty(
+          'docketNumber',
+        );
+      });
+      it('has valid data', () => {
+        const submissionObj = JSON.parse(transform(null, inputObject));
+        expect(
+          submissionObj.additionalData.bankruptcy.hasBeenAdjudicatedBankrupt,
+        ).to.equal(true);
+        expect(submissionObj.additionalData.bankruptcy.dateDischarged).to.equal(
+          '10/2004',
+        );
+        expect(submissionObj.additionalData.bankruptcy.courtLocation).to.equal(
+          'Tampa, FL',
+        );
+        expect(submissionObj.additionalData.bankruptcy.docketNumber).to.equal(
+          '123456',
+        );
+      });
+    });
+  });
+  describe('applicantCertifications', () => {
+    it('has valid structure', () => {
+      const submissionObj = JSON.parse(transform(null, inputObject));
+      expect(submissionObj).haveOwnProperty('applicantCertifications');
+      expect(submissionObj.applicantCertifications).to.be.an('object');
+      expect(submissionObj.applicantCertifications).haveOwnProperty(
+        'veteranSignature',
+      );
+      expect(submissionObj.applicantCertifications).haveOwnProperty(
+        'veteranDateSigned',
+      );
+    });
+    it('has valid data', () => {
+      const submissionObj = JSON.parse(transform(null, inputObject));
+      expect(submissionObj.applicantCertifications.veteranSignature).to.equal(
+        'Greg A Anderson',
+      );
+      expect(submissionObj.applicantCertifications.veteranDateSigned).to.equal(
+        '03/09/2022',
+      );
+    });
+  });
+});
+
+// yarn test:unit --app-folder financial-status-report --log-level=error
```

Comment:
Super minor: Notes should not be kept in code. This might be a great thing to put into a read me